tox.ini

```diff
@@ -31,8 +31,6 @@ commands = {posargs:py.test -v\
            {envsitepackagesdir}/virtualenvrunner {toxinidir}/tests}
 
 [pytest]
-addopts = --pep8
-pep8maxlinelength = 90
```

Comment:
As pep8 is not used any more pylint should be configured to expect same line length at [.pylintrc#L196](https://github.com/nokia/virtualenvrunner/blob/master/.pylintrc#L196)

Reply:
ok